micrometer/src/main/scala/zio/metrics/connectors/micrometer/MicroMeterConfig.scala

```diff
@@ -0,0 +1,6 @@
+package zio.metrics.connectors.micrometer
+
+final case class MicroMeterConfig(summaryPercentileDigitsOfPrecision: Int)
```

Comment:
Very minor but let it be MicrometerConfig :)

Reply:
Done